src/main/java/longah/exception/ExceptionMessage.java

```diff
@@ -15,27 +15,35 @@ public enum ExceptionMessage {
     INVALID_VALUE_FORMAT ("Invalid value format."),
     NO_TRANSACTION_FOUND ("Transaction list is empty."),
     NO_DEBTS_FOUND ("No debts found."),
-    // Search Command Exceptions
-    NO_TRANSACTION_FOUND_FOR_MEMBER ("The queried member does not own any transactions"),
-    NO_DEBTS_FOUND_FOR_MEMBER("The queried member does not owe any debts"),
+    TRANSACTIONS_SUMMED_UP ("No pending payments."),
     // Data Storage Exceptions
     STORAGE_FILE_NOT_FOUND ("File not found."),
     STORAGE_FILE_NOT_CREATED ("File not created."),
     STORAGE_FILE_NOT_READ ("File not read."),
     STORAGE_FILE_NOT_WRITTEN ("File not written."),
     INVALID_STORAGE_CONTENT ("Invalid content in storage file, line ignored."),
+    STORAGE_FILE_CORRUPTED ("Storage file is corrupted."),
 
     // Ui exceptions
-    INVALID_FINDPAYMENT_COMMAND("Invalid command format." +
-            " Use 'findPayment PERSON'"),
-    INVALID_SETTLEUP_COMMAND("Invalid command format." +
+    INVALID_COMMAND ("Invalid command. Use 'add', 'listdebts', 'listtransactions'," +
```

Comment:
Good if the exception message for invalid command are updated to the new standards with command words separated